### ISSUE TITLE
superman64: blender issue?

### ISSUE THREAD

**Issue by olivieryuyu** [gonetz/GLideN64#986]
![sdfdsf](https://cloud.githubusercontent.com/assets/8987490/15229805/e7dee82c-1892-11e6-83d3-3d88963ed08a.jpg)

the white should not be here. The menu has weird white effects on the letters as well when fading.

**Comment by gonetz** [comment]
Can't reproduce it. I guess, you met the same issue as #975
![superman-001](https://cloud.githubusercontent.com/assets/8983092/15244397/dd1ea388-1922-11e6-890c-b0b69acf4daf.png)

**Comment by olivieryuyu** [comment]
i tried many times with different emus.

will test again tonight

**Comment by oddMLan** [comment]
For what it's worth, I've been experiencing the issue you linked more often. I came to the conclusion that deleting the shader cache does not help or alleviate the issue; it still happens (and it's recurrent) with shader storage disabled. It's even recurrent between emulator restarts.

Changing the filtering to 3-point filtering sometimes triggers it. And switching back and forth between that and bilinear filtering sometimes fixes it, other times it makes it worse.
I found that entering fullscreen and getting out of fullscreen does the same thing, sometimes it makes transparencies break, but if you repeat it it sometimes fixes it.
I think it's an OGL issue and something it's not getting cleared.

Resetting settings mid-emulation also fixes transparencies when they are broken, though I haven't tested that a lot, but it has worked for me a couple of times.
